plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduMetadata.java

```diff
@@ -161,14 +161,16 @@ private ConnectorTableMetadata getTableMetadata(KuduTableHandle tableHandle)
     {
         KuduTable table = tableHandle.getTable(clientSession);
         Schema schema = table.getSchema();
+        // Kudu table comments are not null, instead empty strings
+        Optional<String> tableComment = (table.getComment().isEmpty()) ? Optional.empty() : Optional.ofNullable(table.getComment());
```

Comment:
This is just to conform the way Kudu handles null comments to the way Trino handles null comments.

Kudu: `''` => Trino: `null`

Reply:
This is a pretty trivial expression.